### PR TITLE
Add testnet command

### PR DESCRIPTION
**Pull request by alpe**
Adds command to setup a testnet:

```sh
palomad testnet init-files --chain-id=testing --output-dir=./testnet --v=4 --keyring-backend=test --minimum-gas-prices=0.000001ugrain --staking-denom=ugrain --commit-timeout=900ms --single-host --starting-ip-address=""
```

Disclaimer: 
copied from wasmd which came from cosmos-sdk/simapp with some modifications

### CODE REVIEW

File: cmd/palomad/commands.go [hunk 1]
```diff
@@ -111,6 +114,7 @@ func initRootCmd(
 		txCommand(basicManager),
 		keys.Commands(),
 	)
+	wasmcli.ExtendUnsafeResetAllCmd(rootCmd)
```
Comment (alpe) [inline cmd/palomad/commands.go:117]: This clears the `wasm` folder on unsafe-reset-all, too. Not related to the testnet command though

Comment (taariq) [reply cmd/palomad/commands.go:117]: @byte-bandit do we want to include this in paloma?

Comment (taariq) [reply cmd/palomad/commands.go:117]: Yup This is it.

**Comment by taariq** [comment]
@alpe quick lint check?

**Review by taariq — APPROVED** [review]

**Pull request merged**